src/bilder/images/edxapp/templates/forum.env

```diff
@@ -0,0 +1,25 @@
+{{ with secret "mongodb-mitxonline/creds/forum" }}
+export MONGOHQ_URL="mongodb://{{ .Data.username }}:{{ .Data.password }}@mongodb-master.service.consul:27017/forum"
+{{ end }}
+{{ with secret "secret-mitxonline/edx-forum" }}
+export API_KEY="{{ .Data.forum_api_key }}"
+{{ end }}
+export RBENV_ROOT="/edx/app/forum/.rbenv"
+export GEM_HOME="/edx/app/forum/.gem"
+export GEM_PATH="/edx/app/forum/.gem"
+export PATH="/edx/app/forum/cs_comments_service/bin:/edx/app/forum/.rbenv/bin:/edx/app/forum/.rbenv/shims:/edx/app/forum/.gem/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+export RACK_ENV="production"
+export SINATRA_ENV="production"
+export SEARCH_SERVER="http://localhost:9200"
+export SEARCH_SERVER_ES7="http://elasticsearch.service.consul:9200"
+export MONGOID_AUTH_MECH=":scram"
+export HOME="/edx/app/forum"
+export NEW_RELIC_ENABLE="True"
+export NEW_RELIC_APP_NAME="default_env-default_deployment-forum"
```

Comment:
Shouldn't we stick with either underscores or hyphens?

Reply:
That's just a copy/paste from the vanilla Forum env file.